core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/versionmapper/CamelForFuseOnOpenShiftToBomMapper.java

```diff
@@ -29,8 +29,8 @@
 	public static final String FUSE_740_CAMEL_VERSION = "2.21.0.fuse-740039-redhat-00001";
 	public static final String FUSE_750_CAMEL_VERSION = "2.21.0.fuse-750033-redhat-00001";
 	public static final String FUSE_760_CAMEL_VERSION = "2.21.0.fuse-760027-redhat-00001";
-	public static final String FUSE_770_CAMEL_VERSION_CAMEL_2_21 = "2.23.2.fuse-770010-redhat-00001";
```

Comment:
the version was wrong for Camel 2.21 but I think that it needs to be kept and fixed, not removed

Reply:
We discussed that in the chat and you told me its ok to remove it.

Reply:
I was talking about removing it from the list of tested versions

Reply:
spun off another Jira for that: https://issues.redhat.com/browse/FUSETOOLS-3390